shared/LiteCoreJNI.exp

```diff
@@ -0,0 +1,71 @@
+#  LiteCoreJNI.exp
+#  Couchbase Lite Core
+#
+#  Created by Jens Alfke on 9/14/15.
+#  Copyright (c) 2015-2016 Couchbase. All rights reserved.
```

Comment:
The license header is not correct.

Reply:
I'm also wondering if we need this file or not.

Reply:
Good catch.  Totally out of scope for this change.  I will open a separate ticket for it, though.

Reply:
Opened issue: https://github.com/couchbase/couchbase-lite-android/issues/1877